xchanger/main.py

```diff
@@ -28,24 +28,35 @@ def read_microservice_config(config_path):
         return None
 
     microservice_config = Munch(microservice_config)
-    logger.info(microservice_config)
     return microservice_config
 
 
+@retry(stop=stop_after_attempt(10), wait=wait_fixed(5))
+def connect_to_rabbitmq(AMQP_URI):
```

Comment:
Your function argument is shadowing a global variable with the same name.

Reply:
whoops - that should be lower case. Thanks Tom